tasks/python-reference-task/README.md

```diff
@@ -0,0 +1,29 @@
+# @cumulus/python-reference-task
+
+This 'task' is a reference task that is included with Cumulus to allow integration testing of the [`Cumulus message adapter`](https://github.com/nasa/cumulus-message-adapter) with a built python lambda.
+
+## Use
+
+Developmental use of this lambda is intended to be simple - the task returns a static processing output, integration tests can be then built against the `Reference Task` step in the (`PythonReference`)[https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/python_reference_workflow.tf] workflow.
```

Comment:
This is referred to as the `PythonProcess` workflow in the other README. I'm fine with either, just want to be consistent

Reply:
👍  22b5a0612